src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,26 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+
+
+/**
+ * Sorts all persons in the address book before displaying the new sorted list to the user.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Sorts all persons in the address book based on their name in lexicographic order before displaying all persons in the address book as a list with index numbers.\n"
```

Comment:
Try to avoid single lines with more than 110 characters.  A reader has to scroll horizontally which is undesired.